ProjectLighthouse.Servers.GameServer/Middlewares/DigestMiddleware.cs

```diff
@@ -16,7 +16,6 @@ public DigestMiddleware(RequestDelegate next, bool computeDigests) : base(next)
         this.computeDigests = computeDigests;
     }
 
-    #if !DEBUG
```

Comment:
Can we keep this and other related non debug preprocessors

Reply:
Will add these back as soon as I can. This would reintroduce test failures though, how would you recommend to fix those?

Reply:
The tests only fail in debug mode, so it shouldn't be an issue for the CI.